### PR TITLE
update aliases for release-7.1 branch

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

- Remove all aliases for v7.1
- Add aliases for `br/br-checkpoint-backup.md`. `br/br-checkpoint.md` was renamed to `br/br-checkpoint-backup.md` in v7.1.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-checkpoint-backup.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 断点备份
 summary: 了解断点备份功能，包括它的使用场景、实现原理以及使用方法。
-aliases: ["/zh/tidb/dev/br-checkpoint"]
+aliases: ["/tidb/dev/br-checkpoint"]
```
Comment (Oreoxmt) [inline br/br-checkpoint-backup.md:4]:
```suggestion
aliases: ["/zh/tidb/dev/br-checkpoint"]
```

File: br/br-checkpoint-backup.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 断点备份
 summary: 了解断点备份功能，包括它的使用场景、实现原理以及使用方法。
-aliases: ["/zh/tidb/dev/br-checkpoint"]
+aliases: ["/zh/tidb/stable/br-checkpoint"]
```
Comment (ran-huang) [inline br/br-checkpoint-backup.md:4]: I think this alias should be removed.

The url is existent: https://docs.pingcap.com/zh/tidb/stable/br-checkpoint

**Review by ran-huang — APPROVED** [review]

File: br/br-checkpoint-backup.md (hunk 2 above)
Comment (Oreoxmt) [reply br/br-checkpoint-backup.md:4]: `br/br-checkpoint.md` was renamed to `br/br-checkpoint-backup.md` in both master and release-7.1 branches.

Once `release-7.1` becomes the stable version, this URL https://docs.pingcap.com/zh/tidb/stable/br-checkpoint will become unavailable if this alias `/zh/tidb/stable/br-checkpoint` is removed.

Comment (ran-huang) [reply br/br-checkpoint-backup.md:4]: That makes sense. Thanks

**Pull request merged**